ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -159,24 +159,37 @@ message DamlSubmitterInfo {
   string application_id = 3;
 
   reserved 4; // was maximum_record_time
-
-  // Timestamp until which the ledger MUST deduplicate this submission.
-  // Measured against the ledger's clock.
-  google.protobuf.Timestamp deduplicate_until = 5;
+  reserved 5; // was deduplicate_until
+
+  // Duration that determines the range of existing DamlLogEntries against which this
+  // submission deduplicated.
+  //  
+  // The ledger guarantees that for any existing entry e0 the deduplicate_until field of
+  // its associated DamlCommandDedupValue cdv0 and the entry e1 created from this submission
+  // it holds that 
+  //
+  //    if (e1.record_time >= e0.record_time >= e1.record_time - e1.deduplication_duration) AND
+  //       (e0.command_id, e0.application_id, e0.submitters) == 
```

Comment:
Do you use set equality for comparing `_.submitters`? If so, it's worth mentioning here. If not, then we can start another discussion of whether the committer should validate that the `submitters` are given in canonical form (e.g., sorted)

Reply:
Yes, this should be structural equality, as the lookup of CommandDedupKeys is done on their serialized representation.

So, we should start the discussion wrt ensuring that submitters are given in canonical form.

@miklos-da: do current CommandDedupKeys store the submitters in sorted form?

Reply:
Thanks for the confirmation @miklos-da . I have adapted the property in 25722d4 to use set equality in the comparison.